lib/glific/contacts/contact.ex

```diff
@@ -66,6 +68,7 @@ defmodule Glific.Contacts.Contact do
     field :optin_time, :utc_datetime
     field :optout_time, :utc_datetime
     field :last_message_at, :utc_datetime
+    field :last_communication_at, :utc_datetime
```

Comment:
I think we can put a comment here. 

I am assuming this is the time when the last message exchanged (irrespective of inbound or outbound) with the contact.